src/Utility/Language.php

```diff
@@ -13,33 +13,59 @@ class Language {
 	 *
 	 * @return array
 	 */
-	public static function parseLanguageList($languageList = null, $forceLowerCase = true) {
+	public static function parseLanguageList($languageList = null, $forceAllLowerCase = true, $keepDuplicates = true) {
```

Comment:
Renamed this because it is now what it does.
>inoas: also I really dislike the option interface
> inoas: should I make it check for true/false then set forceAllLowerCase - else if is an array check for options if you want to extend it later?